frontend/projects/abtesting/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.ts

```diff
@@ -243,11 +243,24 @@ export class ExperimentDesignComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   validateConditionCodes(conditions: ExperimentCondition[]) {
+
+    let conditionUniqueErrorText = this.translate.instant('home.new-experiment.design.condition-unique-validation.text');
     const conditionCodes = conditions.map(condition => condition.conditionCode);
     if (conditionCodes.length !== new Set(conditionCodes).size) {
-      this.conditionCodeError = this.translate.instant('home.new-experiment.design.condition-unique-validation.text')
-    } else {
-      this.conditionCodeError = null;
+      this.conditionCodeError = conditionUniqueErrorText;
+    }  else {
+        this.conditionCodeError = null;
+    }
+  }
+
+  validatehasConditionCodeDefault(conditions: ExperimentCondition[]) {
```

Comment:
method name should be camelCase. Rename it to validateHasConditionCodeDefault.

Reply:
okay

Reply:
Done